improver/api/__init__.py

```diff
@@ -30,6 +30,7 @@
     "BaseNeighbourhoodProcessing": "improver.nbhood.nbhood",
     "CalculateForecastBias": "improver.calibration.simple_bias_correction",
     "CalibratedForecastDistributionParameters": "improver.calibration.emos_calibration",
+    "calculate_wind_chill_cube": "improver.temperature.feels_like_temperature",
```

Comment:
```suggestion
    "calculate_wind_chill_cube": "improver.temperature.feels_like_temperature",
    "CalibratedForecastDistributionParameters": "improver.calibration.emos_calibration",
```

Alphabetical sorting.